ibis/backends/duckdb/tests/test_client.py

```diff
@@ -420,3 +421,24 @@ def test_memtable_doesnt_leak(con, monkeypatch):
     df = ibis.memtable({"a": [1, 2, 3]}, name=name).execute()
     assert name not in con.list_tables()
     assert len(df) == 3
+
+
+def test_create_table_with_nulls(con):
```

Comment:
Do we expect this behavior in any other backends?

Reply:
In general, no you cannot. Most of our backends do not directly expose a `NULL` type.